vdr/vdr.go

```diff
@@ -91,7 +91,21 @@ func (r *VDR) Configure(_ core.ServerConfig) error {
 
 // Diagnostics returns the diagnostics for this engine
 func (r *VDR) Diagnostics() []core.DiagnosticResult {
```

Comment:
Should we already add diagnostics on one's own conflicted documents, which can then be monitored by an administrator (or preferably, tooling which alerts)

Reply:
Issue added, since multiple PRs are required first: #241